dao.js

```diff
@@ -0,0 +1,41 @@
+var pg = require('pg');
+var client = new pg.Client({
+    user: "lzndoxqpefxjsw",

```

Comment:
We shouldn't store the db connections details in the repo. It should be stored as environment variables in Heroku, i.e. `process.env.DATABASE_URL`.

[Heroku-postgres Connecting in Node](https://devcenter.heroku.com/articles/heroku-postgresql#connecting-in-node-js)